_includes/specials/taproot/en/06-multisignatures.md

```diff
@@ -85,28 +91,30 @@ something like the following:
    (the same as is commonly done now for
    multisigs).
 
-2. The wallet also generates a set of nonces that are also shared with
-   the other participants.  The wallet can generate these nonces using
-   BIP32 hardened derivation.  Nonces are 32 bytes and you need two of
-   them per signature.  For infrequently used wallets, all the nonces
-   needed for the entire wallet lifetime can be shared up front.  For
-   more frequently used wallets (e.g. LN routing nodes), each wallet can
-   send its signature for the current transaction along with its nonces
-   for the next transaction.
-
-3. Any of the wallets can then generate an aggregated public key by
+2. Any of the wallets can then generate an aggregated public key by
    combining its pubkey at a certain BIP32 depth with pubkeys at the
    same depth from all other wallets in the multisignature association.
    The aggregated public key can be used to receive P2TR payments.
 
 4. When one of the wallets wants to spend the funds, it uses a
    [PSBT][topic psbt]-based workflow similar to what it would use with
-   script-based multisig.  Unlike multisig, the wallet uses its next
-   nonces and the next nonces of all the other participants to create a
-   shared nonce according to the MuSig2 algorithm; it then creates a
-   partial signature over that nonce and the transaction.
-
-5. When the other wallets receive the PSBT, they use the same procedure.
+   script-based multisig, but now two rounds of communication between
+   signers are required.  In the first round, the proposer creates the
+   unsigned transaction and includes a pair of randomly-generated
+   nonces.  It's important that the nonces not be derived in an entirely
+   deterministic way that could lead to the same nonce being used again
```

Comment:
This point could be stressed (it's more than just important). There are ways to rationalize ignoring this advice. I've heard for example: "When I generate the nonces deterministically, I obtain the same signature in the end. So why would I have to add randomness". Well you only get the same signature if the other signers generate their nonces deterministically, but they don't have to.

Reply:
Wording changed from "important" to "absolutely essential"